### PR TITLE
Return families' enrolments in the requested session/class

### PR DESCRIPTION
**Pull request by brittanylau**
### Notion ticket link

[Display the enrolled class & status of families in the selected session](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=16114024596844d3b60f80dcc1560a9a)

### Implementation description

- updated the SessionDetailSerializer and ClassDetailSerializer to pass an enrolment context (for the family's enrolment in that session/class) to the FamilySerializer
  - in the FamilySerializer, if an enrolment was specified, return that in the `enrolment` property
- update tests & clean up some old ones
  - some of the test cases for the family serializer overlapped with one another, so just merged them all into one test case

### Steps to test

1. python manage.py test

### What should reviewers focus on?

- code cleanliness

### Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] I have run the linter
- [x] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: registration/tests/serializers/test_family.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 from registration.models import Family, Student
 from enrolments.models import Enrolment, Session, Class
+from enrolments.tests.utils import utils
```
Comment (jolenezheng) [inline registration/tests/serializers/test_family.py]: is this needed :o

File: enrolments/tests/views/test_class.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 from django.urls import NoReverseMatch, reverse
 from rest_framework import status
-from rest_framework.test import APITestCase
+from rest_framework.request import Request
+from rest_framework.test import APITestCase, APIRequestFactory
+import json
```
Comment (jolenezheng) [inline enrolments/tests/views/test_class.py]: is this used in this class? :O

**Review by jolenezheng — APPROVED** [review]
I think there are 2 imports that miiiiiight not be needed? left comments there! but otherwise LGTM

File: registration/tests/serializers/test_family.py (hunk 1 above)
Comment (brittanylau) [reply registration/tests/serializers/test_family.py]: no :') fixed

File: enrolments/tests/views/test_class.py (hunk 2 above)
Comment (brittanylau) [reply enrolments/tests/views/test_class.py]: no :') also fixed

**Pull request merged**